components/transactions/TransactionRejectButton.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from '@apollo/client';
+import { FormattedMessage } from 'react-intl';
+
+import { getErrorFromGraphqlException } from '../../lib/errors/';
+import { GraphQLContext } from '../../lib/graphql/context';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+
+import { Box, Flex } from '../Grid';
+import MessageBox from '../MessageBox';
+import StyledButton from '../StyledButton';
+
+/* eslint-disable graphql/template-strings */
```

Comment:
Why is that required?

Reply:
Er... not sure, I will be honest, I copy-pasted the `TransactionRefundButton` component and then altered it to fit the new purpose 🙈 Will remove this.